src/screens/Profile/ProfileForm.js

```diff
@@ -22,11 +22,16 @@ import React from 'react';
 import { Dimensions } from 'react-native';
 import t from 'tcomb-form-native';
 import styled from 'styled-components/native';
-import TextInput from 'components/TextInput';
 import { Wrapper } from 'components/Layout';
 import { spacing } from 'utils/variables';
 import Button from 'components/Button';
-import { isValidEmail, isValidName, isValidCityName } from 'utils/validators';
+import { InputTemplate, Form } from 'components/ProfileRelatedForm';
```

Comment:
What do you think about renaming `ProfileForm` to `EditProfile` and `components/ProfileRelatedForm` to `components/ProfileForm`?

Reply:
I like it, thanks for that